.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run CCNP E2E test
+on:
+   schedule:
+    - cron: '20 10 * * *'
+   workflow_dispatch:
+jobs:
+  E2E_test:
+    name: Publish ccnp package
+    runs-on: tdvm-ut
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Run Test
+        run: |
+           cd ./test/e2e-test/
+           sudo ./ci-test.sh
```

Comment:
Do we have a notification on results?

Reply:
If this test fails, I will be notified via email.